src/hooks/useLoadAllOnIdle.ts

```diff
@@ -0,0 +1,37 @@
+import { useIdle } from "react-use"
+
+let LOADED = false
+let FETCHING = false
+
+type ManifestFiles = {
+  [key: string]: string
+}
+
+function loadAllStaticAssets() {
+  if (LOADED) return
+  if (FETCHING) return
+  FETCHING = true
+  fetch("/asset-manifest.json")
```

Comment:
how much net io will this be imposing on the user?

Reply:
ca. 10 MB right now

Reply:
example asset-manifest.json

```json
{
  "files": {
    "main.css": "/static/css/main.b6a0a4a9.css",
    "main.js": "/static/js/main.e0602db2.js",
    "static/css/9436.fade48c9.chunk.css": "/static/css/9436.fade48c9.chunk.css",
    "static/js/9436.e320ba56.chunk.js": "/static/js/9436.e320ba56.chunk.js",
    "static/css/423.fade48c9.chunk.css": "/static/css/423.fade48c9.chunk.css",
    "static/js/423.d3c18ff3.chunk.js": "/static/js/423.d3c18ff3.chunk.js",
    "static/js/787.f6c983c7.chunk.js": "/static/js/787.f6c983c7.chunk.js",
    "static/js/7021.edc19d9c.chunk.js": "/static/js/7021.edc19d9c.chunk.js",
    "static/js/6108.81d1149c.chunk.js": "/static/js/6108.81d1149c.chunk.js",
    "static/js/7562.e79040ae.chunk.js": "/static/js/7562.e79040ae.chunk.js",
    "static/js/1066.15d8aa6e.chunk.js": "/static/js/1066.15d8aa6e.chunk.js",
    "static/js/9602.ce4afa7e.chunk.js": "/static/js/9602.ce4afa7e.chunk.js",
    "static/js/7908.372c76a0.chunk.js": "/static/js/7908.372c76a0.chunk.js",
    "static/js/6290.7195ae11.chunk.js": "/static/js/6290.7195ae11.chunk.js",
    "static/js/4783.e0ee22d5.chunk.js": "/static/js/4783.e0ee22d5.chunk.js",
    "static/js/2260.08c644a8.chunk.js": "/static/js/2260.08c644a8.chunk.js",
    "static/js/9528.cb502d23.chunk.js": "/static/js/9528.cb502d23.chunk.js",
    "static/js/2415.267f8eca.chunk.js": "/static/js/2415.267f8eca.chunk.js",
    "static/js/5716.71afc15a.chunk.js": "/static/js/5716.71afc15a.chunk.js",
    "static/js/4183.077f8360.chunk.js": "/static/js/4183.077f8360.chunk.js",
    "static/js/3931.ce61b706.chunk.js": "/static/js/3931.ce61b706.chunk.js",
    "static/js/8489.581a7bd0.chunk.js": "/static/js/8489.581a7bd0.chunk.js",
    "static/js/6365.5891d43f.chunk.js": "/static/js/6365.5891d43f.chunk.js",
    "static/js/7664.13bb9df0.chunk.js": "/static/js/7664.13bb9df0.chunk.js",
    "static/js/2835.e15e1c3a.chunk.js": "/static/js/2835.e15e1c3a.chunk.js",
    "static/js/2475.d8d0f5b7.chunk.js": "/static/js/2475.d8d0f5b7.chunk.js",
    "static/js/5803.0007d0cb.chunk.js": "/static/js/5803.0007d0cb.chunk.js",
    "static/js/4329.c9052948.chunk.js": "/static/js/4329.c9052948.chunk.js",
    "static/js/873.3158920f.chunk.js": "/static/js/873.3158920f.chunk.js",
    "static/js/7501.449c7551.chunk.js": "/static/js/7501.449c7551.chunk.js",
    "static/js/9488.c0db4859.chunk.js": "/static/js/9488.c0db4859.chunk.js",
    "static/js/2202.676cd9a5.chunk.js": "/static/js/2202.676cd9a5.chunk.js",
    "static/js/8717.137a10f9.chunk.js": "/static/js/8717.137a10f9.chunk.js",
    "static/js/5732.2a176d51.chunk.js": "/static/js/5732.2a176d51.chunk.js",
    "static/js/7894.3c01ca35.chunk.js": "/static/js/7894.3c01ca35.chunk.js",
    "static/js/8549.64b4200a.chunk.js": "/static/js/8549.64b4200a.chunk.js",
    "static/js/4172.3df0e339.chunk.js": "/static/js/4172.3df0e339.chunk.js",
    "static/js/4419.7d84cbb9.chunk.js": "/static/js/4419.7d84cbb9.chunk.js",
    "static/js/6885.8f188435.chunk.js": "/static/js/6885.8f188435.chunk.js",
    "static/js/8840.2fa4ea7c.chunk.js": "/static/js/8840.2fa4ea7c.chunk.js",
    "static/js/7327.22e86c5e.chunk.js": "/static/js/7327.22e86c5e.chunk.js",
    "static/js/7175.fa63b6a6.chunk.js": "/static/js/7175.fa63b6a6.chunk.js",
    "static/js/8365.2453f274.chunk.js": "/static/js/8365.2453f274.chunk.js",
    "static/js/9771.711f4fd7.chunk.js": "/static/js/9771.711f4fd7.chunk.js",
    "static/js/9724.cbc98014.chunk.js": "/static/js/9724.cbc98014.chunk.js",
    "static/js/8487.d605a000.chunk.js": "/static/js/8487.d605a000.chunk.js",
    "static/js/5705.6946546e.chunk.js": "/static/js/5705.6946546e.chunk.js",
    "static/js/4163.09a3f542.chunk.js": "/static/js/4163.09a3f542.chunk.js",
    "static/js/9043.6ffbfcdf.chunk.js": "/static/js/9043.6ffbfcdf.chunk.js",
    "static/js/672.5f036523.chunk.js": "/static/js/672.5f036523.chunk.js",
    "static/js/3619.236dcfc4.chunk.js": "/static/js/3619.236dcfc4.chunk.js",
    "static/js/214.88d86861.chunk.js": "/static/js/214.88d86861.chunk.js",
    "static/js/9112.108ef56c.chunk.js": "/static/js/9112.108ef56c.chunk.js",
    "static/js/6566.559e2b1a.chunk.js": "/static/js/6566.559e2b1a.chunk.js",
    "static/js/5504.03d19540.chunk.js": "/static/js/5504.03d19540.chunk.js",
    "static/js/6845.51f297f4.chunk.js": "/static/js/6845.51f297f4.chunk.js",
    "static/js/4218.570c0c69.chunk.js": "/static/js/4218.570c0c69.chunk.js",
    "static/js/834.3fd2673a.chunk.js": "/static/js/834.3fd2673a.chunk.js",
    "static/js/5242.070a6eb2.chunk.js": "/static/js/5242.070a6eb2.chunk.js",
    "static/media/pic_2.jpg": "/static/media/pic_2.ae2bc3942d323586059a.jpg",
    "static/media/pic_4.webp": "/static/media/pic_4.643a5b04a40006f99979.webp",
    "static/media/pic_4.jpg": "/static/media/pic_4.dc9218da4617c5e4eaa5.jpg",
    "static/media/pic_3.jpg": "/static/media/pic_3.84c8e0434c2d32ae3ba1.jpg",
    "static/media/Inter-Bold.ttf": "/static/media/Inter-Bold.88fa7ae373b07b41ecce.ttf",
    "static/media/Inter-SemiBold.ttf": "/static/media/Inter-SemiBold.4d56bb21f2399db8ad48.ttf",
    "static/media/Inter-Medium.ttf": "/static/media/Inter-Medium.6dcbc9bed1ec438907ee.ttf",
    "static/media/Inter-Regular.ttf": "/static/media/Inter-Regular.e89cb19905e7db5591b0.ttf",
    "static/media/SFUIText-Bold.ttf": "/static/media/SFUIText-Bold.39019507036595a661d4.ttf",
    "static/media/SFUIText-Semibold.ttf": "/static/media/SFUIText-Semibold.ecf579529674436eb3c5.ttf",
    "static/media/SFUIText-Medium.ttf": "/static/media/SFUIText-Medium.e04b6fa4dc6937fba636.ttf",
    "static/media/SFUIText-Regular.ttf": "/static/media/SFUIText-Regular.96141c83bd71d7cccb63.ttf",
    "static/media/pic_2.webp": "/static/media/pic_2.dfb4e272bfe8d3f5726f.webp",
    "static/media/bridge-screenshot.jpg": "/static/media/bridge-screenshot.09137f05eb1bdd3f64ef.jpg",
    "static/media/pic_5.jpg": "/static/media/pic_5.fa0d9f60d4f3cf16c32b.jpg",
    "static/media/pic_5.webp": "/static/media/pic_5.5a4b5be80608848c09ea.webp",
    "static/media/pic_1.webp": "/static/media/pic_1.7641495c190227192650.webp",
    "static/media/pic_3.webp": "/static/media/pic_3.b9eab2acddcb82f41198.webp",
    "static/media/pic_1.jpg": "/static/media/pic_1.8ed02a125d5af16b167b.jpg",
    "static/media/bridge-screenshot.webp": "/static/media/bridge-screenshot.a3464561f387d9ff3295.webp",
    "static/media/bridge-screenshot-m.jpg": "/static/media/bridge-screenshot-m.5907ef9b0f8b4c9b6a63.jpg",
    "static/media/bridge-screenshot-m.webp": "/static/media/bridge-screenshot-m.8f5006f6d2036147f1f3.webp",
    "static/media/logo_with_text.png": "/static/media/logo_with_text.7c6cafcac81093d6f83b.png",
    "static/media/EVM-equivalence.png": "/static/media/EVM-equivalence.ff9687bba270f9053f54.png",
    "static/media/PulpDisplay-SemiBold.ttf": "/static/media/PulpDisplay-SemiBold.8f6cd0dfa6079d911a88.ttf",
    "static/media/PulpDisplay-Regular.ttf": "/static/media/PulpDisplay-Regular.57a2a10d3263f053bc0c.ttf",
    "static/media/PulpDisplay-Medium.ttf": "/static/media/PulpDisplay-Medium.eeee1fcfd037a8e728b7.ttf",
    "static/media/proofGeneration.md": "/static/media/proofGeneration.b7cfaf5720ecdf507c47.md",
    "static/media/Ye.jpg": "/static/media/Ye.30b30319158496779aba.jpg",
    "static/media/Sandy.jpg": "/static/media/Sandy.ca59458f886503274e6c.jpg",
    "static/media/rocket.png": "/static/media/rocket.79d4096de2b0cf64bd6a.png",
    "static/media/zkEVM.md": "/static/media/zkEVM.24771ff90e057befe55a.md",
    "static/media/Haichen.jpg": "/static/media/Haichen.9ea15b5547886c86d834.jpg",
    "static/media/Ye.webp": "/static/media/Ye.1cecaf2bffa2b3ec3eec.webp",
    "static/media/Sandy.webp": "/static/media/Sandy.38aff98bb282238687f6.webp",
    "static/media/kzg.md": "/static/media/kzg.7d29c0c09b6774578aa1.md",
    "static/media/Haichen.webp": "/static/media/Haichen.f17034e73c8c056f85fe.webp",
    "static/media/rocket.webp": "/static/media/rocket.6212676819996f7d4576.webp",
    "static/media/technicalPrinciples.md": "/static/media/technicalPrinciples.537a68205351c08b540d.md",
    "static/media/architecture.md": "/static/media/architecture.c6d713682038a3f7c9ef.md",
    "static/media/visionAndValues.md": "/static/media/visionAndValues.8baf1adf7c4cba1485a0.md",
    "static/media/EVM-equivalence.webp": "/static/media/EVM-equivalence.9947b6d2520eb902ea33.webp",
    "static/media/preAlphaTestnet.md": "/static/media/preAlphaTestnet.06ffee291323eb3c8b6e.md",
    "static/media/upgradingPreAlphaTestnet.md": "/static/media/upgradingPreAlphaTestnet.a2c200f12c7995b1be6a.md",
    "static/media/scalability.webp": "/static/media/scalability.bedfb7dede0ef7f6f380.webp",
    "static/media/alphaTestnet.md": "/static/media/alphaTestnet.5ea0576ea097c352a4d5.md",
    "static/media/website.svg": "/static/media/website.e9949f3d5ca9dbaefc74a25b5095e431.svg",
    "static/media/copy.svg": "/static/media/copy.358299dfd1ac6814adcda7f05602651c.svg",
    "static/media/security.webp": "/static/media/security.ebcb990e5dec3a5ab793.webp",
    "static/media/releaseNote0109.md": "/static/media/releaseNote0109.08e9ec09365e8151ede9.md",
    "static/media/twitter.svg": "/static/media/twitter.99c00c923e91bd191cc7d3a146652da4.svg",
    "index.html": "/index.html",
    "static/media/eth.svg": "/static/media/eth.38bb58fcc572178e85531ffe5579dba5.svg",
    "static/media/transfer.svg": "/static/media/transfer.6a7654c145f995443bd99cb2290f1f8e.svg",
    "static/media/exit.svg": "/static/media/exit.b77851ba2272efd398408bcc4a2cadec.svg",
    "static/media/arrow-down.svg": "/static/media/arrow-down.b748f8a338c401b9e098fa21996398a0.svg",
    "main.b6a0a4a9.css.map": "/static/css/main.b6a0a4a9.css.map",
    "main.e0602db2.js.map": "/static/js/main.e0602db2.js.map",
    "9436.fade48c9.chunk.css.map": "/static/css/9436.fade48c9.chunk.css.map",
    "9436.e320ba56.chunk.js.map": "/static/js/9436.e320ba56.chunk.js.map",
    "423.fade48c9.chunk.css.map": "/static/css/423.fade48c9.chunk.css.map",
    "423.d3c18ff3.chunk.js.map": "/static/js/423.d3c18ff3.chunk.js.map",
    "787.f6c983c7.chunk.js.map": "/static/js/787.f6c983c7.chunk.js.map",
    "7021.edc19d9c.chunk.js.map": "/static/js/7021.edc19d9c.chunk.js.map",
    "6108.81d1149c.chunk.js.map": "/static/js/6108.81d1149c.chunk.js.map",
    "7562.e79040ae.chunk.js.map": "/static/js/7562.e79040ae.chunk.js.map",
    "1066.15d8aa6e.chunk.js.map": "/static/js/1066.15d8aa6e.chunk.js.map",
    "9602.ce4afa7e.chunk.js.map": "/static/js/9602.ce4afa7e.chunk.js.map",
    "7908.372c76a0.chunk.js.map": "/static/js/7908.372c76a0.chunk.js.map",
    "6290.7195ae11.chunk.js.map": "/static/js/6290.7195ae11.chunk.js.map",
    "4783.e0ee22d5.chunk.js.map": "/static/js/4783.e0ee22d5.chunk.js.map",
    "2260.08c644a8.chunk.js.map": "/static/js/2260.08c644a8.chunk.js.map",
    "9528.cb502d23.chunk.js.map": "/static/js/9528.cb502d23.chunk.js.map",
    "2415.267f8eca.chunk.js.map": "/static/js/2415.267f8eca.chunk.js.map",
    "5716.71afc15a.chunk.js.map": "/static/js/5716.71afc15a.chunk.js.map",
    "4183.077f8360.chunk.js.map": "/static/js/4183.077f8360.chunk.js.map",
    "3931.ce61b706.chunk.js.map": "/static/js/3931.ce61b706.chunk.js.map",
    "8489.581a7bd0.chunk.js.map": "/static/js/8489.581a7bd0.chunk.js.map",
    "6365.5891d43f.chunk.js.map": "/static/js/6365.5891d43f.chunk.js.map",
    "7664.13bb9df0.chunk.js.map": "/static/js/7664.13bb9df0.chunk.js.map",
    "2835.e15e1c3a.chunk.js.map": "/static/js/2835.e15e1c3a.chunk.js.map",
    "2475.d8d0f5b7.chunk.js.map": "/static/js/2475.d8d0f5b7.chunk.js.map",
    "5803.0007d0cb.chunk.js.map": "/static/js/5803.0007d0cb.chunk.js.map",
    "4329.c9052948.chunk.js.map": "/static/js/4329.c9052948.chunk.js.map",
    "873.3158920f.chunk.js.map": "/static/js/873.3158920f.chunk.js.map",
    "7501.449c7551.chunk.js.map": "/static/js/7501.449c7551.chunk.js.map",
    "9488.c0db4859.chunk.js.map": "/static/js/9488.c0db4859.chunk.js.map",
    "2202.676cd9a5.chunk.js.map": "/static/js/2202.676cd9a5.chunk.js.map",
    "8717.137a10f9.chunk.js.map": "/static/js/8717.137a10f9.chunk.js.map",
    "5732.2a176d51.chunk.js.map": "/static/js/5732.2a176d51.chunk.js.map",
    "7894.3c01ca35.chunk.js.map": "/static/js/7894.3c01ca35.chunk.js.map",
    "8549.64b4200a.chunk.js.map": "/static/js/8549.64b4200a.chunk.js.map",
    "4172.3df0e339.chunk.js.map": "/static/js/4172.3df0e339.chunk.js.map",
    "4419.7d84cbb9.chunk.js.map": "/static/js/4419.7d84cbb9.chunk.js.map",
    "6885.8f188435.chunk.js.map": "/static/js/6885.8f188435.chunk.js.map",
    "8840.2fa4ea7c.chunk.js.map": "/static/js/8840.2fa4ea7c.chunk.js.map",
    "7327.22e86c5e.chunk.js.map": "/static/js/7327.22e86c5e.chunk.js.map",
    "7175.fa63b6a6.chunk.js.map": "/static/js/7175.fa63b6a6.chunk.js.map",
    "8365.2453f274.chunk.js.map": "/static/js/8365.2453f274.chunk.js.map",
    "9771.711f4fd7.chunk.js.map": "/static/js/9771.711f4fd7.chunk.js.map",
    "9724.cbc98014.chunk.js.map": "/static/js/9724.cbc98014.chunk.js.map",
    "8487.d605a000.chunk.js.map": "/static/js/8487.d605a000.chunk.js.map",
    "5705.6946546e.chunk.js.map": "/static/js/5705.6946546e.chunk.js.map",
    "4163.09a3f542.chunk.js.map": "/static/js/4163.09a3f542.chunk.js.map",
    "9043.6ffbfcdf.chunk.js.map": "/static/js/9043.6ffbfcdf.chunk.js.map",
    "672.5f036523.chunk.js.map": "/static/js/672.5f036523.chunk.js.map",
    "3619.236dcfc4.chunk.js.map": "/static/js/3619.236dcfc4.chunk.js.map",
    "214.88d86861.chunk.js.map": "/static/js/214.88d86861.chunk.js.map",
    "9112.108ef56c.chunk.js.map": "/static/js/9112.108ef56c.chunk.js.map",
    "6566.559e2b1a.chunk.js.map": "/static/js/6566.559e2b1a.chunk.js.map",
    "5504.03d19540.chunk.js.map": "/static/js/5504.03d19540.chunk.js.map",
    "6845.51f297f4.chunk.js.map": "/static/js/6845.51f297f4.chunk.js.map",
    "4218.570c0c69.chunk.js.map": "/static/js/4218.570c0c69.chunk.js.map",
    "834.3fd2673a.chunk.js.map": "/static/js/834.3fd2673a.chunk.js.map",
    "5242.070a6eb2.chunk.js.map": "/static/js/5242.070a6eb2.chunk.js.map"
  },
  "entrypoints": [
    "static/css/main.b6a0a4a9.css",
    "static/js/main.e0602db2.js"
  ]
}
```

Reply:
on-disk size, unpressed
```
[ 192]  .
├── [ 256]  css
│   ├── [ 11K]  423.fade48c9.chunk.css
│   ├── [ 12K]  423.fade48c9.chunk.css.map
│   ├── [ 11K]  9436.fade48c9.chunk.css
│   ├── [ 12K]  9436.fade48c9.chunk.css.map
│   ├── [ 16K]  main.b6a0a4a9.css
│   └── [ 11K]  main.b6a0a4a9.css.map
├── [3.7K]  js
│   ├── [1.0K]  1066.15d8aa6e.chunk.js
│   ├── [1.1K]  1066.15d8aa6e.chunk.js.map
│   ├── [1.6K]  214.88d86861.chunk.js
│   ├── [1.6K]  214.88d86861.chunk.js.map
│   ├── [1.9K]  2202.676cd9a5.chunk.js
│   ├── [2.0K]  2202.676cd9a5.chunk.js.map
│   ├── [2.7K]  2260.08c644a8.chunk.js
│   ├── [2.7K]  2260.08c644a8.chunk.js.map
│   ├── [ 701]  2415.267f8eca.chunk.js
│   ├── [ 769]  2415.267f8eca.chunk.js.map
│   ├── [1.5K]  2475.d8d0f5b7.chunk.js
│   ├── [1.6K]  2475.d8d0f5b7.chunk.js.map
│   ├── [ 849]  2835.e15e1c3a.chunk.js
│   ├── [ 909]  2835.e15e1c3a.chunk.js.map
│   ├── [4.9K]  3619.236dcfc4.chunk.js
│   ├── [5.2K]  3619.236dcfc4.chunk.js.map
│   ├── [1.8K]  3931.ce61b706.chunk.js
│   ├── [2.0K]  3931.ce61b706.chunk.js.map
│   ├── [1.4K]  4163.09a3f542.chunk.js
│   ├── [1.5K]  4163.09a3f542.chunk.js.map
│   ├── [ 897]  4172.3df0e339.chunk.js
│   ├── [ 975]  4172.3df0e339.chunk.js.map
│   ├── [7.5K]  4183.077f8360.chunk.js
│   ├── [7.9K]  4183.077f8360.chunk.js.map
│   ├── [ 26K]  4218.570c0c69.chunk.js
│   ├── [ 85K]  4218.570c0c69.chunk.js.map
│   ├── [1.3K]  423.d3c18ff3.chunk.js
│   ├── [3.1K]  423.d3c18ff3.chunk.js.map
│   ├── [1.7K]  4329.c9052948.chunk.js
│   ├── [1.8K]  4329.c9052948.chunk.js.map
│   ├── [10.0K]  4419.7d84cbb9.chunk.js
│   ├── [ 10K]  4419.7d84cbb9.chunk.js.map
│   ├── [ 611]  4783.e0ee22d5.chunk.js
│   ├── [ 674]  4783.e0ee22d5.chunk.js.map
│   ├── [250K]  5242.070a6eb2.chunk.js
│   ├── [515K]  5242.070a6eb2.chunk.js.map
│   ├── [116K]  5504.03d19540.chunk.js
│   ├── [ 811]  5504.03d19540.chunk.js.LICENSE.txt
│   ├── [553K]  5504.03d19540.chunk.js.map
│   ├── [1.2K]  5705.6946546e.chunk.js
│   ├── [1.3K]  5705.6946546e.chunk.js.map
│   ├── [5.3K]  5716.71afc15a.chunk.js
│   ├── [5.5K]  5716.71afc15a.chunk.js.map
│   ├── [ 18K]  5732.2a176d51.chunk.js
│   ├── [ 19K]  5732.2a176d51.chunk.js.map
│   ├── [ 825]  5803.0007d0cb.chunk.js
│   ├── [ 889]  5803.0007d0cb.chunk.js.map
│   ├── [5.9K]  6108.81d1149c.chunk.js
│   ├── [6.1K]  6108.81d1149c.chunk.js.map
│   ├── [ 951]  6290.7195ae11.chunk.js
│   ├── [1023]  6290.7195ae11.chunk.js.map
│   ├── [3.4K]  6365.5891d43f.chunk.js
│   ├── [3.5K]  6365.5891d43f.chunk.js.map
│   ├── [2.3K]  6566.559e2b1a.chunk.js
│   ├── [2.4K]  6566.559e2b1a.chunk.js.map
│   ├── [8.1K]  672.5f036523.chunk.js
│   ├── [8.2K]  672.5f036523.chunk.js.map
│   ├── [ 54K]  6845.51f297f4.chunk.js
│   ├── [198K]  6845.51f297f4.chunk.js.map
│   ├── [ 599]  6885.8f188435.chunk.js
│   ├── [ 670]  6885.8f188435.chunk.js.map
│   ├── [4.1K]  7021.edc19d9c.chunk.js
│   ├── [4.3K]  7021.edc19d9c.chunk.js.map
│   ├── [1.3K]  7175.fa63b6a6.chunk.js
│   ├── [1.4K]  7175.fa63b6a6.chunk.js.map
│   ├── [7.4K]  7327.22e86c5e.chunk.js
│   ├── [7.6K]  7327.22e86c5e.chunk.js.map
│   ├── [3.0K]  7501.449c7551.chunk.js
│   ├── [3.1K]  7501.449c7551.chunk.js.map
│   ├── [2.7K]  7562.e79040ae.chunk.js
│   ├── [2.7K]  7562.e79040ae.chunk.js.map
│   ├── [ 527]  7664.13bb9df0.chunk.js
│   ├── [ 597]  7664.13bb9df0.chunk.js.map
│   ├── [4.5K]  787.f6c983c7.chunk.js
│   ├── [ 10K]  787.f6c983c7.chunk.js.map
│   ├── [2.4K]  7894.3c01ca35.chunk.js
│   ├── [2.5K]  7894.3c01ca35.chunk.js.map
│   ├── [ 22K]  7908.372c76a0.chunk.js
│   ├── [ 22K]  7908.372c76a0.chunk.js.map
│   ├── [2.8M]  834.3fd2673a.chunk.js
│   ├── [2.0K]  834.3fd2673a.chunk.js.LICENSE.txt
│   ├── [ 11M]  834.3fd2673a.chunk.js.map
│   ├── [2.5K]  8365.2453f274.chunk.js
│   ├── [2.6K]  8365.2453f274.chunk.js.map
│   ├── [3.2K]  8487.d605a000.chunk.js
│   ├── [3.5K]  8487.d605a000.chunk.js.map
│   ├── [1.0K]  8489.581a7bd0.chunk.js
│   ├── [1.1K]  8489.581a7bd0.chunk.js.map
│   ├── [6.9K]  8549.64b4200a.chunk.js
│   ├── [7.0K]  8549.64b4200a.chunk.js.map
│   ├── [2.6K]  8717.137a10f9.chunk.js
│   ├── [2.7K]  8717.137a10f9.chunk.js.map
│   ├── [1.6K]  873.3158920f.chunk.js
│   ├── [1.7K]  873.3158920f.chunk.js.map
│   ├── [3.2K]  8840.2fa4ea7c.chunk.js
│   ├── [3.4K]  8840.2fa4ea7c.chunk.js.map
│   ├── [2.4K]  9043.6ffbfcdf.chunk.js
│   ├── [2.5K]  9043.6ffbfcdf.chunk.js.map
│   ├── [ 889]  9112.108ef56c.chunk.js
│   ├── [ 959]  9112.108ef56c.chunk.js.map
│   ├── [1.9K]  9436.e320ba56.chunk.js
│   ├── [5.3K]  9436.e320ba56.chunk.js.map
│   ├── [ 20K]  9488.c0db4859.chunk.js
│   ├── [ 20K]  9488.c0db4859.chunk.js.map
│   ├── [2.0K]  9528.cb502d23.chunk.js
│   ├── [2.2K]  9528.cb502d23.chunk.js.map
│   ├── [7.4K]  9602.ce4afa7e.chunk.js
│   ├── [7.5K]  9602.ce4afa7e.chunk.js.map
│   ├── [2.2K]  9724.cbc98014.chunk.js
│   ├── [2.3K]  9724.cbc98014.chunk.js.map
│   ├── [5.1K]  9771.711f4fd7.chunk.js
│   ├── [5.4K]  9771.711f4fd7.chunk.js.map
│   ├── [559K]  main.e0602db2.js
│   ├── [2.2K]  main.e0602db2.js.LICENSE.txt
│   └── [2.5M]  main.e0602db2.js.map
└── [1.8K]  media
    ├── [7.0K]  EVM-equivalence.9947b6d2520eb902ea33.webp
    ├── [ 46K]  EVM-equivalence.ff9687bba270f9053f54.png
    ├── [ 24K]  Haichen.9ea15b5547886c86d834.jpg
    ├── [ 18K]  Haichen.f17034e73c8c056f85fe.webp
    ├── [309K]  Inter-Bold.88fa7ae373b07b41ecce.ttf
    ├── [307K]  Inter-Medium.6dcbc9bed1ec438907ee.ttf
    ├── [303K]  Inter-Regular.e89cb19905e7db5591b0.ttf
    ├── [308K]  Inter-SemiBold.4d56bb21f2399db8ad48.ttf
    ├── [ 40K]  PulpDisplay-Medium.eeee1fcfd037a8e728b7.ttf
    ├── [ 41K]  PulpDisplay-Regular.57a2a10d3263f053bc0c.ttf
    ├── [ 42K]  PulpDisplay-SemiBold.8f6cd0dfa6079d911a88.ttf
    ├── [298K]  SFUIText-Bold.39019507036595a661d4.ttf
    ├── [295K]  SFUIText-Medium.e04b6fa4dc6937fba636.ttf
    ├── [285K]  SFUIText-Regular.96141c83bd71d7cccb63.ttf
    ├── [296K]  SFUIText-Semibold.ecf579529674436eb3c5.ttf
    ├── [ 22K]  Sandy.38aff98bb282238687f6.webp
    ├── [ 29K]  Sandy.ca59458f886503274e6c.jpg
    ├── [ 23K]  Ye.1cecaf2bffa2b3ec3eec.webp
    ├── [ 31K]  Ye.30b30319158496779aba.jpg
    ├── [2.4K]  alphaTestnet.5ea0576ea097c352a4d5.md
    ├── [9.1K]  architecture.c6d713682038a3f7c9ef.md
    ├── [ 260]  arrow-down.b748f8a338c401b9e098fa21996398a0.svg
    ├── [101K]  bridge-screenshot-m.5907ef9b0f8b4c9b6a63.jpg
    ├── [ 57K]  bridge-screenshot-m.8f5006f6d2036147f1f3.webp
    ├── [233K]  bridge-screenshot.09137f05eb1bdd3f64ef.jpg
    ├── [135K]  bridge-screenshot.a3464561f387d9ff3295.webp
    ├── [1.7K]  copy.358299dfd1ac6814adcda7f05602651c.svg
    ├── [ 592]  eth.38bb58fcc572178e85531ffe5579dba5.svg
    ├── [ 509]  exit.b77851ba2272efd398408bcc4a2cadec.svg
    ├── [ 20K]  kzg.7d29c0c09b6774578aa1.md
    ├── [ 56K]  logo_with_text.7c6cafcac81093d6f83b.png
    ├── [156K]  pic_1.7641495c190227192650.webp
    ├── [152K]  pic_1.8ed02a125d5af16b167b.jpg
    ├── [578K]  pic_2.ae2bc3942d323586059a.jpg
    ├── [270K]  pic_2.dfb4e272bfe8d3f5726f.webp
    ├── [309K]  pic_3.84c8e0434c2d32ae3ba1.jpg
    ├── [152K]  pic_3.b9eab2acddcb82f41198.webp
    ├── [390K]  pic_4.643a5b04a40006f99979.webp
    ├── [329K]  pic_4.dc9218da4617c5e4eaa5.jpg
    ├── [173K]  pic_5.5a4b5be80608848c09ea.webp
    ├── [198K]  pic_5.fa0d9f60d4f3cf16c32b.jpg
    ├── [4.9K]  preAlphaTestnet.06ffee291323eb3c8b6e.md
    ├── [ 35K]  proofGeneration.b7cfaf5720ecdf507c47.md
    ├── [1.5K]  releaseNote0109.08e9ec09365e8151ede9.md
    ├── [ 11K]  rocket.6212676819996f7d4576.webp
    ├── [ 29K]  rocket.79d4096de2b0cf64bd6a.png
    ├── [3.0K]  scalability.bedfb7dede0ef7f6f380.webp
    ├── [1.7K]  security.ebcb990e5dec3a5ab793.webp
    ├── [9.5K]  technicalPrinciples.537a68205351c08b540d.md
    ├── [ 527]  transfer.6a7654c145f995443bd99cb2290f1f8e.svg
    ├── [1.1K]  twitter.99c00c923e91bd191cc7d3a146652da4.svg
    ├── [4.6K]  upgradingPreAlphaTestnet.a2c200f12c7995b1be6a.md
    ├── [8.5K]  visionAndValues.8baf1adf7c4cba1485a0.md
    ├── [2.3K]  website.e9949f3d5ca9dbaefc74a25b5095e431.svg
    └── [ 24K]  zkEVM.24771ff90e057befe55a.md

3 directories, 176 files
```